assets/src/edit-story/components/thumbnail/thumbnail.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+/**
+ * Internal dependencies
+ */
+import { noop, Icons } from '../../../design-system';
+import { THUMBNAIL_TYPES, THUMBNAIL_SCRIM_CLASSNAME } from './constants';
+import { Container, Background, NestedIconContainer, Scrim } from './styles';
+
+const includeDefaultScrimBackground = [
+  THUMBNAIL_TYPES.SHAPE,
+  THUMBNAIL_TYPES.TEXT,
+];
+
+/**
+ * Element and page thumbnails, used in the prepublish checklist
+ *
+ * @param {Node} displayBackground Node that renders the element. Relies on PagePreview or LayerIcon (getDefinitionForType, see panels/design/layer) to keep element rendering consistent with carousel and layer panels. See /storybook for demo.
+ * @param {boolean} isError Thumbnail errors don't prevent further interaction with the thumbnail, they just change the presentation so user knows the action failed.
```

Comment:
I'm wondering if eslint is complaining cus these are all supposed to be object properties

Reply:
you're right! updated to be `props.isError` and all is well.